addons/web-support/api/src/main/java/org/jboss/windup/web/addons/websupport/tsmodelgen/TypeScriptModelsGeneratorConfig.java

```diff
@@ -1,79 +1,91 @@
-package org.jboss.windup.web.addons.tsmodelsgen;
+package org.jboss.windup.web.addons.websupport.tsmodelgen;
 
 import java.nio.file.Path;
 
 /**
+ * Defines how the TS files will be generated.
+ *
  * @author <a href="mailto:zizka@seznam.cz">Ondrej Zizka</a>
  */
 public class TypeScriptModelsGeneratorConfig
 {
-    public static enum AdjacencyMode { PROXIED, MATERIALIZED, MIXED, DECORATED; }
-    public static enum FileNamingStyle { CAMELCASE, LOWERCASE_DASHES, LOWERCASE_DOTS; }
-    
```

Comment:
Hmm I left these inside... I think that's all right since it is for the purpose of this config. Isn't that nicer?